pytorch/helm/values.yaml

```diff
@@ -0,0 +1,29 @@
+#    Deployment options and parameters
+#    Set these to your cloud vendor and your instance size
+cloud: demo
+instance: demo
+
+#    Container image to use (including architecture)
+#    - quay.io/kinvolk/pytorch-benchmark:v1-amd64
+#    - quay.io/kinvolk/pytorch-benchmark:v1-arm64
+image: quay.io/kinvolk/pytorch-benchmark:v1-amd64
```

Comment:
Why not push the image with generic (platform agnostic) tag and use that and let docker decide which image to pull?